packages/@webex/plugin-meetings/src/roap/turnDiscovery.ts

```diff
@@ -202,6 +202,13 @@ export default class TurnDiscovery {
    * @returns {Promise}
    */
   doTurnDiscovery(meeting, isReconnecting) {
+    const isAnyClusterReachable = meeting.webex.meetings.reachability.isAnyClusterReachable();
+
+    if (isAnyClusterReachable) {
+      LoggerProxy.logger.info('Roap:turnDiscovery#doTurnDiscovery --> reachability has not failed, skipping it');
```

Comment:
very minor thing, but the log messages says "reachability has not failed, skipping it" - that suggests that it's skipping reachability, it should probably say "reachability has not failed, skipping TURN discovery"